stellarphot/settings/custom_widgets.py

```diff
@@ -1016,8 +1016,9 @@ def __init__(
         self.photometry_notebook_name = photometry_notebook_name
         self.fitsopen = FitsOpener(
             title=(
-                "Choose any image in the folder of images to do photometry on that "
-                "contains the object of interest"
+                "Choose an image of the object of interest to you.</br>Photometry "
+                "will be performed on all images of that object in the folder "
+                "the object is in."
```

Comment:
OK, this still seems a little oddly phrased, what about:
```suggestion
                "Choose an image of the object of interest to you.</br>Photometry "
                 "will be performed on all images of that object in the folder containing "
                 "the selected image."
```